ui/admin-portal/src/app/model/candidate-field-info.ts

```diff
@@ -63,7 +65,9 @@ export class CandidateFieldInfo {
     let value;
     // Need to format field with the candidate object not value
     if (this.fieldPath === "ieltsScore" || this.fieldPath === "frenchAssessmentScoreNclc" ||
-      this.fieldPath === "latestIntake" || this.fieldPath === "latestIntakeDate") {
+      this.fieldPath === "latestIntake" || this.fieldPath === "latestIntakeDate" ||
+      this.fieldPath === "candidateOpportunities.nextStep" ||
```

Comment:
I know that it is just text "candidateOpportunities" - plural. This doesn't make sense to me. There is no "nextStep" for candidateOpportunities - there is a different next step for each opportunity.

I can't see in this PR how candidateOpportunities.nextStep is processed. Is that in another PR?

Reply:
Because both these fields use a formatter, the field path is never used — but you're right, it's confusing. I've amended to null instead, which works fine. The next step processing is in this PR.

The existing code for using or disregarding the field path is a little hacky `CandidateFieldInfo.getValue()` - perhaps something we could mark for future attention. I think the formatter has been repurposed for something that its name doesn't quite describe.

Reply:
This is sorted now, per our discussion